payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/DropdownItemSpec.kt

```diff
@@ -3,10 +3,12 @@ package com.stripe.android.ui.core.elements
 import android.os.Parcelable
 import androidx.annotation.RestrictTo
 import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.Serializable
 
-@Parcelize
+@Serializable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Parcelize
 data class DropdownItemSpec(
-    val api_value: String?,
-    val display_text: String
+    val api_value: String? = null,
+    val display_text: String = "Other"
```

Comment:
nit: should "Other" use translated string resource?

Reply:
For all the LPMs that use a dropdown today the dropdown text is not translated.  There are a couple of different ways this might change in the future:
1. Have the ability to pass down localized strings
2. LPM has dropdown items that must be translated.

So this works for what we have today and we can adjust in the future.